docs/src/mbtiles.md

```diff
@@ -0,0 +1,10 @@
+# Working with MBTiles archives
+
+We offer the `mbtiles` utility to interact with the `*.mbtiles` files (see [this page](mbtiles-schema.md) for more information) from the command line.
+It allows users to [examine](mbtiles-meta.md), [copy](mbtiles-copy.md), [validate](mbtiles-validation.md) or [compare and apply diffs between them](mbtiles-diff.md).
```

Comment:
```suggestion

It allows users to [examine](mbtiles-meta.md), [copy](mbtiles-copy.md), [validate](mbtiles-validation.md) or [compare and apply diffs between them](mbtiles-diff.md).
```

Reply:
I prefer leaving them together.
I think both are part of the short "one sentence" (here two, as much content) summary, that a good paragrah should have.

Reply:
My bad. I didn't noticed they are one single paragraph but with to two lines.